### PR TITLE
Cloning an Existing Pantheon Site

### LINKED ISSUE #1830: Cloning an Existing Pantheon Site Doc Update
Re: [Cloning an Existing Pantheon Site](https://pantheon.io/docs/clone-site)
## Issue Description:

Needs to be tested, but would like to add `terminus drush ard --destination=sites/default/files/RANDOM-HASH.tgz` as an option for Drupal sites:

For Drupal sites, follow the instructions shown when "Migrating Existing Site" to prepare your archive with drush to use `drush ard`, but use `terminus drush ard` if you're cloning an existing Pantheon site.

If your archive is larger than 256MB, then it will fail to write to the Pantheon filesystem, use the manual method

If your database and code compressed are less than 256MB you can exclude the files directory and upload those separately. 

 `drush ard   --tar-options="--exclude=sites/default/files" `

### PR DESCRIPTION
**Pull request by rachelwhitton**
Closes #1830

## Effect
PR includes the following changes:
- Instruct WordPress users to follow guided migration path (BlogVault)
- Instruct Drupal users to create archives for existing sites using terminus
 - Include troubleshooting section for archives exceeding 256MB

@ari-gold - tested, results matched expected behavior outlined in the original issue. Can you and @nataliejeremy review?

### CODE REVIEW

**Comment by nataliejeremy** [comment]
+1 Looks great!

File: source/_docs/clone-site.md [hunk 1]
```diff
@@ -5,8 +5,47 @@ categories: [developing]
...
+For more information see: [Migrate to Pantheon: WordPress](/docs/migrate-wordpress)
+
+## Drupal
+After selecting **[Migrate Existing Site](https://dashboard.pantheon.io/sites/migrate/)**, you'll create a new Pantheon site. During the **Create Site Archive** step of the migration process, use [Terminus](/docs/terminus) to create an archive of your existing Pantheon site:
+
+```bash
+terminus drush 'ard --destination=sites/default/files/cC1j4cP0GWaMWrD.tgz'
+```
+
+This operation writes the archive to Pantheon's filesystem in a web accessible location (e.g. `http://env-site-name.pantheonsite.io/sites/default/files/cC1j4cP0GWaMWrD.tgz`). Click **Continue Migration** and follow all remaining instructions within the guided migration process.
```
Comment (ari-gold) [inline source/_docs/clone-site.md]: Did you successfully clone Dev and a non-dev environment? Just wondering if we should recommend they clone from Dev?

Would like to include a placeholder for the filename here and below, something like ` --destination=sites/default/files/<RANDOM_HASH>.tgz` ?

**Comment by ari-gold** [comment]
@rachelwhitton Left one comment. Looks great! Thank you!!

File: source/_docs/clone-site.md [hunk 2]
```diff
@@ -5,8 +5,47 @@ categories: [developing]
...
+
+If your database and code compressed are less than 256MB you can exclude the files directory from export using the following steps, otherwise [manually migrate](/docs/migrate-manual) the site.
+
+<ol><li>Use <a href="/docs/terminus">Terminus</a> and the <code>--tar-options</code> flag:<br><br>
+<pre><code class="bash hljs">terminus drush <span class="hljs-string">'ard <b>--tar-options="--exclude=sites/default/files"</b> --destination=sites/default/files/cC1j4cP0GWaMWrD.tgz'</span></code></pre></li></ol>
+
+2. Click **Continue Migration**.
+3. Provide the web accessible URL for your site archive (e.g. `http://env-site-name.pantheonsite.io/sites/default/files/cC1j4cP0GWaMWrD.tgz`) and select **Import Archive**.
+4. Once the migration is complete, click **Visit the Site Dashboard**.
+5. Create an archive of your site's files directory if you have not done so already:
```
Comment (rachelwhitton) [inline source/_docs/clone-site.md]: Is this the method you would suggest for creating an archive of the files dir @ari-gold? I thought about linking to [Export an Existing Drupal Site (Export Files)](https://pantheon.io/docs/drupal-export/#export-files) for context, but decided against it in favor of simpler instructions. Thoughts?

Comment (ari-gold) [inline source/_docs/clone-site.md]: Looks good. Might also mention rsync as an alternative.

Comment (ari-gold) [inline source/_docs/clone-site.md]: Oh wait...they can create a backup of files "Backups" tab and just download that archive...then use dashboard to import, or decompress locally and rsync

**Pull request merged**